### PR TITLE
Adds navigation in dropdown menus in surveys and posts

### PR DESCRIPTION
**Pull request by renujain31**
This pull request makes the following changes:
- It makes the dropdown menu  of posts and surveys accessible via keyboard

Testing checklist:
- [ ] Login your account
- [ ] Go to the survey page
- [ ] The dropdown menu represented by three dots on the page is accessible.

- [ ] Now go to Data section.
- [ ] The dropdown menu represented by three dots alongside posts is also accessible.

- [x] I certify that I ran my checklist

Recordings:
rec1
rec2

Fixes ushahidi/platform# .

Linked platform pattern library pull request: ushahidi/platform-pattern-library#271
Ping @ushahidi/platform

### CODE REVIEW

File: app/main/posts/views/share/post-share.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<button ng-class="{ 'button' : isButton(), 'button-flat': isAdd()}" ng-click="openShareMenu()">
+<button ng-class="{ 'button' : isButton(), 'button-flat': isAdd()}" class="button-survey" ng-click="openShareMenu()">
```
Comment (Angamanga) [inline app/main/posts/views/share/post-share.html:1]: I think this is the reason for the share-button has changed appearance?

**Review by Angamanga — CHANGES_REQUESTED** [review]
This works fine, just remove the new class on the share-button and this is good to go to QA :)

**Review by Angamanga — DISMISSED** [review]
Works fine now! Thank you 🎉

**Comment by Angamanga** [comment]
@Obadha2 Ready for QA 🎉

**Comment by Angamanga** [comment]
@Angamanga Resolve conflicts!

**Review by tuxpiper — APPROVED** [review]
Stamping approval

**Comment by Angamanga** [comment]
@Obadha2 Ready for QA 🎉

**Comment by Obadha2** [comment]
@Angamanga this doesn't quite pass.
1. The flow while tabbing down the Survey, seems to go back and forth. Could this be streamlined? (See attached GIF) 
chrome-capture

2. Options presented in some instances the Delete Survey option appears different from the Duplicate option. (See attached GIF) 
3.  The Delete Survey option is skipped when tabbing.

**Pull request merged**